### PR TITLE
chore: Specify CI build pool name

### PR DESCRIPTION
**Pull request by matmerr**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Review by vakalapa — DISMISSED** [review]
Build pipeline changes look good. You will need to check others for CNM test

File: cni/network/network_test.go [hunk 1]
```diff
@@ -62,12 +59,22 @@ func TestPlugin(t *testing.T) {
 	epInfo := &network.EndpointInfo{
 		IPAddresses: []net.IPNet{*addr},
 	}
-	plugin.nm.CreateEndpoint(nwCfg.Name, epInfo)
+	err = plugin.nm.CreateEndpoint(nwCfg.Name, epInfo)
+	/*
+		if err != nil {
+			t.Fatal(err)
+		}*/
```
Comment (tamilmani1989) [inline cni/network/network_test.go:66]: why this is commented?

File: cni/network/network_test.go [hunk 2]
```diff
@@ -62,12 +59,22 @@ func TestPlugin(t *testing.T) {
...
-}
\ No newline at end of file
+
+	err = plugin.nm.CreateNetwork(nwInfo)
+	/*
+		if err != nil {
+			t.Fatal(err)
+		}*/
+
+	//plugin.Add(args)
```
Comment (tamilmani1989) [inline cni/network/network_test.go:79]: same as above

File: network/network_linux.go [hunk 3]
```diff
@@ -319,14 +319,17 @@ func applyDnsConfig(extIf *externalInterface, ifName string) error {
...
-		if err != nil {
-			return err
+		if setDnsList != "" {
+			cmd := fmt.Sprintf("systemd-resolve --interface=%s%s", ifName, setDnsList)
+			_, err = platform.ExecuteCommand(cmd)
+			if err != nil {
+				return err
+			}
+
+			cmd = fmt.Sprintf("systemd-resolve --interface=%s --set-domain=%s", ifName, extIf.DNSInfo.Suffix)
```
Comment (tamilmani1989) [inline network/network_linux.go:329]: make it separate 
if extIf.DNSInfo.Suffix != "" {
}

File: cnm/network/network_test.go [hunk 4]
```diff
@@ -176,21 +216,68 @@ func TestCreateNetwork(t *testing.T) {
...
 		NetworkID:  networkID,
 		EndpointID: endpointID,
 		Interface:  &remoteApi.EndpointInterface{Address: anySubnet},
 	}
 
+	info.Options = make(map[string]interface{})
+	info.Options["com.docker.network.generic"] = make(map[string]interface{})
+	info.Options["com.docker.network.generic"].(map[string]interface{})[modeOption] = "bridge"
+	info.Options["com.docker.network.generic"].(map[string]interface{})[epInfoOption] = epInfo
+
```
Comment (tamilmani1989) [inline cnm/network/network_test.go:280]: is this still needed?

**Review by tamilmani1989 — CHANGES_REQUESTED** [review]

**Review by tamilmani1989 — DISMISSED** [review]
lgtm

**Review by tamilmani1989 — APPROVED** [review]

**Pull request merged**